src/test/suite/fixtures/recorded/languages/python/clearBranch14.yml

```diff
@@ -0,0 +1,34 @@
+languageId: python
+command:
+  version: 3
+  spokenForm: clear branch
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: branch}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    try:
+      pass
+    except* ValueError as eg:
+      pass
+    else:
+      pass
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  marks: {}
+finalState:
+  documentContents: |-
+
```

Comment:
It feels a bit weird for `branch` to work on the `try` block itself 🤷

Reply:
Yeah I'm not entirely sure what the use case for that is tbh 😅, but seemed consistent, and maybe someone would want to use it with the `copy` action?